### ISSUE TITLE
Import error: No module named theme

### ISSUE THREAD

**Issue by mtribaldos** [viewflow/django-material#225]
There is a problem with the instantiation of a theme app, according to the documentation, in the following way:
```
INSTALLED_APPS = (
    # Material apps
    'material.theme.blue',
    'material',
    'material.admin',
    'material.frontend',
    ...
```
It seems the file `material/theme/__init__.py` in the "theme" package definition is missing.

**Comment by kmmbvnr** [comment]
yeah, py 2.7  bug

**Comment by kmmbvnr** [comment]
I've released the fixed version `0.12.2` and pro.
